fxola/Banka
# #165515777  Implement realtime notification on transactions functionality

#### What does this PR do?
- Implement real-time notification on transactions functionality

#### Description of Task to be completed?
- install @sendgrid/mail to enable sending emails from web app
- write sendemail method to handle logic for sending mail
- write findAccountowner method in account model to get user email to
send notifications to.
- delivers[#165515777]

#### How should this be manually tested?
- Clone the repository using ```git clone https://github.com/fxola/Banka.git .``` in your terminal
- run ```npm install``` on your terminal
- sign up on the platform using the endpoint  POST ```api/v1/auth/signup``` 
- create a bank account on the platform using the endpoint POST ```api/v1/accounts```
- login in as an admin or staff using the POST ```api/v1/auth/signin```  endpoint  to perform transactions
on the newly created account using the credentials below
```
{
 "email":"user@admin.com",
 "password":"adminuser"
}
```
- perform a transaction on the new account created using the POST ```api/v1/transactions/<account-number>``` endpoint

- check inbox/spam of email address provided for transaction notification 


#### Any background context you want to provide?
- refer to the env.example file for configuration details
- get and add SENDGRID_API_KEY='yourapikey' to your env variable from sendgrid's website

#### What are the relevant pivotal tracker stories?
- PT Story link - [#165515777] (https://www.pivotaltracker.com/n/projects/2321791/stories/165515777)

#### Screenshots (if appropriate)
- ![one](https://user-images.githubusercontent.com/26928860/56632243-c2047680-6650-11e9-902f-fbc636cd88d5.PNG)

#### Questions:
- N/A
